primitives/src/util/tests/prep_db.rs

```diff
@@ -11,9 +11,10 @@ lazy_static! {
         ids.insert("leader", "awesomeLeader".into());
         ids.insert("follower", "awesomeFollower".into());
         ids.insert("user", "awesomeTestUser".into());
-        ids.insert("publisher", "myAwesomePublisher".into());
+        ids.insert("publisher", "b7d3f81e857692d13e9d63b232a90f4a1793189e".into());
```

Comment:
I've added real Hashes taken from `Jerry`, since Rust is typesafe and explodes when you pass anything that is invalid to the ABI crate for Address, FixedBytes and etc.

Reply:
👍